GameServerLib/Logic/GameObjects/Spell.cs

```diff
@@ -295,11 +297,16 @@ public virtual bool cast(float x, float y, Unit u = null, uint futureProjNetId =
             return true;
         }
 
-        private void RunCastLua()
+        
+        private void RunCast()
```

Comment:
That name seems ugly. Something like `RunCastScripts` would be better in my opinion.